src/components/warning_message/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { withTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+
+import YiviAppBar from '../../widgets/yivi_app_bar';
+import YiviButton from '../../widgets/yivi_button';
+import Column from '../../widgets/column';
+import Spacer from '../../widgets/spacer';
+
+const mapStateToProps = (state) => {
+  return {
+    explanation: state.login.explanation,
+    details: state.login.details,
+  };
+};
+
+// TODO: Maybe convert WarninMessage and ErrorMessage into a single component.
+const WarningMessage = (props) => {
```

Comment:
`WarninMessage` should be `WarningMessage`

Sidenote: do you want to do something with this TODO?